charts/ingress-nginx/README.md

```diff
@@ -4,280 +4,108 @@
-`defaultBackend.podSecurityContext` | Security context policies to add to the default backend | `{}`
-`defaultBackend.service.annotations` | annotations for default backend service | `{}`
-`defaultBackend.service.clusterIP` | internal default backend cluster service IP (set to `"-"` to pass an empty value) | `nil`
-`defaultBackend.service.omitClusterIP` | (Deprecated) To omit the `clusterIP` from the default backend service | `false`
-`defaultBackend.service.externalIPs` | default backend service external IP addresses | `[]`
-`defaultBackend.service.loadBalancerIP` | IP address to assign to load balancer (if supported) | `""`
-`defaultBackend.service.loadBalancerSourceRanges` | list of IP CIDRs allowed access to load balancer (if supported) | `[]`
-`defaultBackend.service.type` | type of default backend service to create | `ClusterIP`
-`defaultBackend.serviceAccount.create` | if `true`, create a backend service account. Only useful if you need a pod security policy to run the backend. | `true`
-`defaultBackend.serviceAccount.name` | The name of the backend service account to use. If not set and `create` is `true`, a name is generated using the fullname template. Only useful if you need a pod security policy to run the backend. | ``
-`imagePullSecrets` | name of Secret resource containing private registry credentials | `nil`
-`rbac.create` | if `true`, create & use RBAC resources | `true`
-`rbac.scope` | if `true`, do not create & use clusterrole and -binding. Set to `true` in combination with `controller.scope.enabled=true` to disable load-balancer status updates and scope the ingress entirely. | `false`
-`podSecurityPolicy.enabled` | if `true`, create & use Pod Security Policy resources | `false`
-`serviceAccount.create` | if `true`, create a service account for the controller | `true`
-`serviceAccount.name` | The name of the controller service account to use. If not set and `create` is `true`, a name is generated using the fullname template. | ``
-`revisionHistoryLimit` | The number of old history to retain to allow rollback. | `10`
-`tcp` | TCP service key:value pairs. The value is evaluated as a template. | `{}`
-`udp` | UDP service key:value pairs The value is evaluated as a template. | `{}`
-
-These parameters can be passed via Helm's `--set` option
-```console
-$ helm install ingress-nginx --name my-release \
-    --set controller.metrics.enabled=true
-```
+By default the ingress-nginx controller has service interruptions whenever it's pods are restarted or redeployed. In order to fix that, see the excellent blog post by Lindsay Landry from Codecademy: [Kubernetes: Nginx and Zero Downtime in Production](https://medium.com/codecademy-engineering/kubernetes-nginx-and-zero-downtime-in-production-2c910c6a5ed8).
 
-Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+### Migrating from stable/nginx-ingress
```

Comment:
:+1:

Reply:
Just making sure, does thumbs up mean you think this section is good enough to deprecate the `stable/nginx-ingress` chart? I think the sooner there aren't two locations for users the better. We can still work to improve this, but just checking how you feel for this stage of the plan to officially deprecate stable and move here?

Reply:
This section looks reasonable and clear about what to expect